src/cdk/__tests__/index.ts

```diff
@@ -10,12 +11,16 @@ describe('The rule that warns on every NagSuppressions.addResourceSuppressions()
   const notEdited = 'some/nested/not-edited'
   const includePaths = ['some-dir', edited, notEdited]
   const filteredPaths = R.map((folder) => `${folder}/index.ts`, includePaths)
+  const [, editedFilePath] = filteredPaths
```

Comment:
@quesabe is it a typo? How does this work?

Reply:
It is intended. We're just taking the second element of the array, which is `some/nested/edited/index.ts`. Previously this path was handwritten in tests but I decided to store it as a variable.